### ISSUE TITLE
Need yeoman generator

### ISSUE THREAD

**Issue by bsommardahl** [dwyl/hapi-typescript-example#4]

**Comment by nelsonic** [comment]
![image](https://cloud.githubusercontent.com/assets/194400/26518684/40dce16e-42ad-11e7-95b1-f774d46115fd.png)

@bsommardahl could you please elaborate on this issue...?
are you asking if `yeoman` is required for this example to work?

**Comment by vforv** [comment]
I agree it would be great if there is generator to generate file structure like for user or tasks... It would save a lot of time during dev.

**Comment by Talento90** [comment]
I have never done a generator! Could someone recommend me a good sources to learn how to build one?

Cheers 👍

**Comment by vforv** [comment]
Check http://yeoman.io/codelab/setup.html
